### PR TITLE
added mapper hack for bv smt issue

### PR DESCRIPTION
**Pull request by rdaly525**
This should be in master until we come up with a way to unbind an instruction to its family

### CODE REVIEW

File: peak/mapper/mapper.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def _convert_io_types(peak_io):
 
 def gen_mapping(
         peak_class : Peak,
-        isa : tp.Type[ISABuilder],
+        bv_isa : tp.Type[BitVector],
```
Comment (rsetaluri) [inline peak/mapper/mapper.py]: These are not right type annotations right? They're still `ISABuilder` objects.

File: peak/mapper/mapper.py [hunk 2]
```diff
@@ -79,22 +80,23 @@ def gen_mapping(
         return
     def f_fun(inst):
         return all(constraint(inst) for constraint in constraints)
+    bv_isa_list = list(filter(f_fun, bv_isa.enumerate()))
```
Comment (rsetaluri) [inline peak/mapper/mapper.py]: Maybe just a comment about why we're keeping two lists, and that its a hack that should go away once we figure out how to do the conversions.

File: peak/mapper/mapper.py (hunk 1 above)
Comment (rdaly525) [reply peak/mapper/mapper.py]: Fixed

File: peak/mapper/mapper.py (hunk 2 above)
Comment (rdaly525) [reply peak/mapper/mapper.py]: Added comment ont he bv_isa argument

**Pull request merged**